### PR TITLE
Added twitter link at the bottom of the landing page. Closes #19

### LINKED ISSUE #19: Add Twitter link at the bottom of the landing page (@datatau_net)

### PR DESCRIPTION
**Pull request by tairosonloa**

### CODE REVIEW

File: templates/common/base.html [hunk 1]
```diff
@@ -80,7 +80,8 @@ <h1 class="hnname"><a href="/">DataTau</a></h1>
               <a href="https://thegurus.tech">Made with ❤️ by The Gurus</a> |
               <a href="mailto:datatau@thegurus.tech">Support</a> |
               <a href="mailto:datatau@thegurus.tech">Contact</a> |
-              <a href="{% url 'latest_feed' %}">RSS</a>
+              <a href="{% url 'latest_feed' %}">RSS</a> |
+              <a rel="noopener noreferrer" target="__blank" href="https://twitter.com/datatau_net">Twitter</a>
```
Comment (ravimashru) [inline templates/common/base.html]: I see that the value of target is `__blank` (two underscores). Is that intentional? AFAIK, it's supposed to be `_blank` (one underscore).

**Review by ravimashru — COMMENTED** [review]
Thank you so much for the contribution! I've left a comment about the `target` attribute. It would be great if you can check it out. :)

**Comment by tairosonloa** [comment]
> Thank you so much for the contribution! I've left a comment about the `target` attribute. It would be great if you can check it out. :)

I'm sorry for the time it took to me to answer you. It was a busy week. That double underscore isn't intentional, it was a typo from my side. However, I have tested it and consecutive clicks on the links always open a new tab (and not only a new tab the first time), so it seems like it's ignoring one of the underscore.

**Comment by ravimashru** [comment]
@tairosonloa  I just tested `__blank` on Chrome v77.0 on MacOS and unfortunately, it opens a new tab only the first time. :(
It would be better if we stick to the standard `_blank`. :)

**Comment by tairosonloa** [comment]
That's weird @mashruravi, I tested it on Linux and Chromium 76.0 and worked. But I agree with you, it would be better if we stick to the standard `_blank`. I'll add a new commit to this branch

**Review by ravimashru — APPROVED** [review]
LGTM!

**Pull request merged**